informant

```diff
@@ -285,6 +285,9 @@ def run():
         feed = CACHE['feed']
     else:
         feed = feedparser.parse(ARCH_NEWS)
+        if feed.bozo:
+            print(feed.bozo_exception)
```

Comment:
Could you update this to use the `err_print` function? It does the same thing just prints to stderr instead of stdout.

Reply:
Done!